ale_linters/terraform/tflint.vim

```diff
@@ -9,23 +9,67 @@ call ale#Set('terraform_tflint_executable', 'tflint')
 
 function! ale_linters#terraform#tflint#Handle(buffer, lines) abort
     let l:output = []
+    let l:pattern = '\v^(.*):(\d+),(\d+)-(\d+)?,?(\d+): (.{-1,}); (.+)$'
+    let l:json = ale#util#FuzzyJSONDecode(a:lines, {})
 
-    for l:error in ale#util#FuzzyJSONDecode(a:lines, [])
-        if l:error.type is# 'ERROR'
-            let l:type = 'E'
-        elseif l:error.type is# 'NOTICE'
-            let l:type = 'I'
-        else
-            let l:type = 'W'
-        endif
-
-        call add(l:output, {
-        \   'lnum': l:error.line,
-        \   'text': l:error.message,
-        \   'type': l:type,
-        \   'code': l:error.detector,
-        \})
-    endfor
+    if type(l:json) is v:t_list
```

Comment:
Maybe add a comment explaining that one branch is for 0.11+ and one is for older version(s).

Reply:
My bad, I totally intended to do that 😅